files/.prettierignore

```diff
@@ -1,25 +1,9 @@
-# unconventional js
-/blueprints/*/files/
```

Comment:
these files are mostly from the app blueprint, and don't match anything that the monorepo root would have